lib/email.ex

```diff
@@ -12,8 +12,8 @@ defmodule Healthlocker.Email do
   def send_reset_email(to_email, token) do
     new_email()
     |> to(to_email)
-    |> from("healthlocker.test@gmail.com")
+    |> from(System.get_env("FROM_EMAIL"))
     |> subject("Healthlocker Reset Password Instructions")
-    |> text_body("Please visit https://www.healthlocker.uk/password/#{token}/edit to reset your password")
+    |> text_body("Please visit https://staging.healthlocker.uk/password/#{token}/edit to reset your password")
```

Comment:
change to `staging` url should be temporary ... (_i.e. and environment variable_)